src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Hierarchy.cs

```diff
@@ -174,8 +175,9 @@ internal static bool IsOverride(Solution solution, ISymbol member, ISymbol symbo
         /// <param name="projects">The projects to search. Can be null to search the entire solution.</param>
         /// <param name="cancellationToken"></param>
         /// <returns>The derived types of the symbol. The symbol passed in is not included in this list.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public static Task<IEnumerable<INamedTypeSymbol>> FindDerivedClassesAsync(
-            INamedTypeSymbol type, Solution solution, IImmutableSet<Project> projects = null, CancellationToken cancellationToken = default)
+            INamedTypeSymbol type, Solution solution, IImmutableSet<Project> projects, CancellationToken cancellationToken)
         {
```

Comment:
existing API.  we're making EB(never) to encourage use of the new form that follow the same pattern as everything else.  This is neither a binary nor source breaking change.